src/main.js

```diff
@@ -45,8 +44,25 @@ let mainWindow;
 let willQuitApp = false;
 
 function createWindow() {
+	// Open the app at the same screen position and size as last time, if possible
+	let windowLayout = { width: 800, height: 600, titleBarStyle: 'hidden-inset' };
+	const previousLayout = userConfig.get('windowLayout');
+	const displaySize = electron.screen.getPrimaryDisplay().workAreaSize;
```

Comment:
I think this is slightly off. I am guessing that `workAreaSize` excludes the menu bar (like `-[NSScreen visibleFrame]`). The symptom is that when my window is touching the very bottom of the screen, relaunching the app goes back to the default position. I'm not sure what the best behavior would be — ideally Electron would let you hook into standard NSWindow frame autosave behavior, but I'm not sure if that's possible.